.travis.yml

```diff
@@ -18,6 +18,7 @@ addons:
   apt:
     sources:
       - ubuntu-toolchain-r-test
+      - llvm-toolchain-trusty-5.0      
```

Comment:
This configuration runs Clang 5.0.0 too:
* https://github.com/ingenue/hunter/blob/3834d1dff2af8424b65887b51f7ffdfab3870c01/.travis.yml#L12-L24

Reply:
okay, so if we update to gcc-7 toolchains we can get clang-5 without an additional installation step?

Reply:
It seems so. Path to Clang will be `/usr/local/clang-5.0.0/bin/clang` (set to PATH by Travis) and you can use `clang-cxx17` toolchain.